build.gradle

```diff
@@ -1,27 +1,28 @@
 buildscript {
     ext {
```

Comment:
Not really related, but decided to bump kotlin & junit versions
Also formatted this file to spaces vs tabs...